src/api/app/models/bs_request.rb

```diff
@@ -154,6 +154,9 @@ def self.new_from_xml(xml)
     if hashed['id']
       theid = hashed.delete('id') { raise 'not found' }
       theid = Integer(theid)
+      if BsRequest.find_by_number(theid)
```

Comment:
so, how would it look alike to get the same? (http error code and OBS xml status error with code and details)